src/app/app.css

```diff
@@ -1,3 +1,7 @@
 html, body, #root {
   height: 100%;
 }
+
+.pf-c-content small {
```

Comment:
This works for sure, but what this is doing is styling any `<small>` in a `.pf-c-content` element. The easier way to do this (and one of the reasons we have these vars) is just to declare the var override using the component selector - no need to write out the elements, classes, etc to override styles. You can just theme any of the main component vars under a single selector. For example:

```
.pf-c-content {
  --pf-c-content--small--Color: red; /* changes all <small> color to red */
  --pf-c-content--blockquote--BorderLeftColor: purple; /* changes all <blockquote> left border color to purple */
  --pf-c-content--hr--BackgroundColor: lemonchiffon; */ changes a <hr> color to lemonchiffon */
}
```

As opposed to writing out:

```
.pf-c-content small { --pf-c-content--small--Color: red; }
.pf-c-content blockquote { --pf-c-content--blockquote--BorderLeftColor: purple; }
.pf-c-content hr { --pf-c-content--hr--BackgroundColor: lemonchiffon; }
```

Another way to demonstrate overrides would be to add a class to your content element `<div class="pf-c-content my-app-custom-content>`, then use `.my-app-custom-content` as the CSS selector instead of `.pf-c-content`, which will only apply the CSS overrides to a content component with that custom class (as opposed to all content components). I guess it depends on the goal of the example override as to whether you want to scope it to a custom class/override or apply the overrides globally.